essence-of-live-coding/test/Util.hs

```diff
@@ -81,3 +81,18 @@ instance (Eq b, Show b) => Testable (CellSimulation a b) where
     , output1 = output
     , output2 = []
     }
+
+-- | Basic unit test for 'Cell' identities.
+--   Check whether one cell behaves the same as another cell.
+data CellIdentitySimulation a b = CellIdentitySimulation
+  { cella :: Cell Identity a b
+  , cellb :: Cell Identity a b
+  , commonInput :: [a]  
+  }
+
+instance (Eq b, Show b) => Testable (CellIdentitySimulation a b) where
+  property CellIdentitySimulation { .. }
+    = let 
```

Comment:
Even simpler you can do something like

```haskell
= property CellSimulation { input1 = input, input2 = input, .. }
```

Reply:
But CellSimulation only has cell, input and output . I don't see how the instance can be created from either CellSimulation or CellMigrationSimulation, can it ?